### ISSUE TITLE
The discrepancy in the reproduction of a certain indicator is too large

### ISSUE THREAD

**Issue by alala521** [sosppxo/MDIN#12]
<img width="781" alt="1" src="https://github.com/user-attachments/assets/a578e739-1665-4111-856e-4ae737cf02b4">
<img width="592" alt="2" src="https://github.com/user-attachments/assets/727712eb-503c-43a1-aff0-c2c90cfbca51">
I used the code provided by the project exactly as it was, without making any modifications. However, in the reproduced results, the indicator for "zt w/ dis" was significantly lower than the results reported in the paper.

**Comment by sosppxo** [comment]
We have successfully reproduced results similar to those reported in the paper multiple times across different GPUs, such as the 3090 and A800. In response to your feedback, we suggest the following: First, please ensure that you are able to reproduce the reported results using the weights we have provided. Secondly, the screenshots you provided do not appear to be from the test script, so please confirm that you have conducted sufficient training (approximately 40-50 epochs) and are using the best.pth weights for testing.

**Comment by alala521** [comment]
> We have successfully reproduced results similar to those reported in the paper multiple times across different GPUs, such as the 3090 and A800. In response to your feedback, we suggest the following: First, please ensure that you are able to reproduce the reported results using the weights we have provided. Secondly, the screenshots you provided do not appear to be from the test script, so please confirm that you have conducted sufficient training (approximately 40-50 epochs) and are using the best.pth weights for testing.

First of all, thank you for your answer! Firstly, I used your weights to test the results and found that they were consistent with the paper. Secondly, I showed the result at 70 epochs, but I changed the batch size from 2 to 4 in the code, while keeping everything else unchanged. I'm not sure if that is the reason

**Comment by sosppxo** [comment]
Thank you for providing the details. Based on our experience, the performance degradation you observed is very likely due to the adjustment of the batch size. Adjusting only the batch size can significantly affect the gradients, which in turn impacts model training and convergence. Therefore, we recommend using the default hyperparameter settings for training. If adjustments are necessary, please adapt all relevant hyperparameters in tandem (such as batch size, learning rate, schedule, etc.). This may require multiple experiments to find the optimal configuration.
